pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -37,8 +37,8 @@ def test_earth_relief_01d_igpp_synbath(data_source):
     assert data.shape == (181, 361)
     npt.assert_allclose(data.lat, np.arange(-90, 91, 1))
     npt.assert_allclose(data.lon, np.arange(-180, 181, 1))
-    npt.assert_allclose(data.min(), -8600.5)
-    npt.assert_allclose(data.max(), 5559.0)
+    npt.assert_allclose(data.min(), -8600.5, atol=0.5)
+    npt.assert_allclose(data.max(), 5559.0, atol=0.5)
```

Comment:
Did you get the uncertainty for the earth_relief grids from the paper(s)? I'm not seeing them on https://www.generic-mapping-tools.org/remote-datasets/earth-relief.html#technical-information.